### DISCUSSION TITLE
[FEATURE-REQUEST] ¿Se puede extender el tamaño del question exec de un componente?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#82] [3. Ideas & Feature Requests] [answered]
Hola

Quisiera saber si está considerado aumentar el espacio del question exec de una question, debido a que es muy pequeño y no se puede trabajar bien
![image](https://user-images.githubusercontent.com/61419471/114940795-4f2dd080-9e10-11eb-92f2-b2b7d05dff3a.png)

Gracias :)

**Comment by acedward** [comment, marked as answer]
Hola, en el presente te recomiendo escribir en un IDE y aquí solo pegar el resultado.
No está en el roadmap de corto plazo mejorar esta interfaz, pero vamos a mejorarlo eventualmente.

**Reply by NicolasPLG** [reply]
gracias @acedward por al ayuda
